packages/atomic/src/components/facets-v1/facet-value-box-color/facet-value-box-color.pcss

```diff
@@ -0,0 +1,11 @@
+@import '../facet-value-box/facet-value-box.pcss';
+
+/**
+ * @prop --atomic-facet-color-boxes-per-row: Number of facet values for the Color Facet to display per row, when the display is 'box'
+ * @prop --atomic-facet-color-boxes-gap: Gap value for facet values for the Color Facet , when the display is 'box'
+ */
+.box-color-container {
```

Comment:
if we reuse the box-container, this file wont be necessary

Reply:
I wanted to make sure the default number of values per row was 3, whereas it's 4 in the box-container class from facet-value-box.pcss